addons/web_editor/static/src/js/editor/rte.js

```diff
@@ -507,6 +507,15 @@ var RTEWidget = Widget.extend({
      * @param {jQuery} $editable
      */
     _enableEditableArea: function ($editable) {
+        if ($editable.data('oe-type') === "datetime" || $editable.data('oe-type') === "date") {
+            debugger;
+            var $linkedFieldNodes = $('#wrapwrap [data-oe-id="' + $editable.data('oe-id') + '"][data-oe-field="' + $editable.data('oe-field') + '"][data-oe-model="' + $editable.data('oe-model') + '"]').not($editable);
+            $linkedFieldNodes.addClass('o_editable_date_field_linked');
+            if (!$editable.hasClass('o_editable_date_field_format_changed')) {
+                $linkedFieldNodes.add($editable).html($editable.data('oe-original-with-format'));
+                $linkedFieldNodes.add($editable).addClass('o_editable_date_field_format_changed');
```

Comment:
Don't repeat the `add` operation twice. In fact not sure why you need it at all: is excluding `$editable` of `$linkedFieldNodes` useful?

Reply:
I don't want the `o_editable_date_field_linked` class on the element you clicked, but that's just my preference, I don't really think having it or not is changing something UX wise.
But I reversed the code, I'll let you see, the `.not` only used where needed.